supertokens_python/recipe/session/exceptions.py

```diff
@@ -51,3 +51,21 @@ def __init__(self, msg: str, clear_cookies: bool = True):
 
 class TryRefreshTokenError(SuperTokensSessionError):
     pass
+
+
+class InvalidClaimsError(SuperTokensSessionError):
+    def __init__(self, msg: str, payload: List[ClaimValidationError]):
+        super().__init__(msg)
+        self.payload = [
+            p.__dict__ for p in payload
+        ]  # Must be JSON serializable as it will be used in response
```

Comment:
> Why are we doing this? 
`ClaimValidationError` is an python object and we cannot directly pass it as JSON while returning back the response. 
doing `p.__dict__` for each one of them converts them into json objects. (You get a dictionary of attributes)

> Just for checking if it's all JSON serializable?
No it's not checking anything. it converts python objects into dictionaries.

>  And if not, will this throw an error?
No. To the best of my knowledge, It can't throw error. Even an empty python object can return `{}`.